### ISSUE TITLE
App crashes when clicking app icon while video in mini-player

### ISSUE THREAD

**Issue by dinmukhamedm** [libre-tube/LibreTube#3703]
### Steps to reproduce

1. Open any video
2. Minimize the app clicking "home" button, or click "pop-up" button
3. Click app icon in the apps menu or home screen

### Expected behavior

App opens the video page with the player open. Same as when clicking the diagonal arrows button in the miniplayer.

### Actual behavior

Upon clicking on the app icon once: playback stops
Upon clicking twice: opens home screen and shows me the exception details

```
android.content.ActivityNotFoundException: Unable to find explicit activity class {com.github.libretube/com.github.libretube.ui.activities.MainActivity}; have you declared this activity in your AndroidManifest.xml?
 at android.app.Instrumentation.checkStartActivityResult(Instrumentation.java:2098)
 at android.app.Instrumentation.execStartActivity(Instrumentation.java:1747)
 at android.app.Activity.startActivityForResult(Activity.java:5473)
 at androidx.activity.ComponentActivity.startActivityForResult(ComponentActivity.java:2)
 at android.app.Activity.startActivityForResult(Activity.java:5431)
 at androidx.activity.ComponentActivity.startActivityForResult(ComponentActivity.java:1)
 at android.app.Activity.startActivity(Activity.java:5817)
 at android.app.Activity.startActivity(Activity.java:5770)
 at com.github.libretube.ui.activities.MainActivity.loadIntentData(MainActivity.kt:32)
 at com.github.libretube.ui.activities.MainActivity.onNewIntent(MainActivity.kt:7)
 at android.app.Activity.performNewIntent(Activity.java:8306)
 at android.app.Instrumentation.callActivityOnNewIntent(Instrumentation.java:1429)
 at android.app.Instrumentation.callActivityOnNewIntent(Instrumentation.java:1442)
 at android.app.ActivityThread.deliverNewIntents(ActivityThread.java:4317)
 at android.app.ActivityThread.handleNewIntent(ActivityThread.java:4324)
 at android.app.servertransaction.NewIntentItem.execute(NewIntentItem.java:56)
 at android.app.servertransaction.ActivityTransactionItem.execute(ActivityTransactionItem.java:45)
 at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:135)
 at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:95)
 at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2443)
 at android.os.Handler.dispatchMessage(Handler.java:106)
 at android.os.Looper.loopOnce(Looper.java:226)
 at android.os.Looper.loop(Looper.java:313)
 at android.app.ActivityThread.main(ActivityThread.java:8751)
 at java.lang.reflect.Method.invoke(Native Method)
 at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:571)
 at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1135)
```

### LibreTube version

0.14.1

### Android version

Android 12

### Other details

Samsung SM-G970N.
This bug was there in 0.14.0 too.

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I will fill out all of the requested information in this form.

**Comment by FireMasterK** [comment]
I couldn't replicate this issue on a `Galaxy S22 Plus` with Android 12. Can you give better instructions to reproduce this? Maybe see if you can reproduce the issue on another device.

**Comment by dinmukhamedm** [comment]
Not that I did anything special, just the 3 steps mentioned in the issue. 

Just tried it on Samsung SM-A530F (Samsung Galaxy A8 2018) with Android 9, and could not reproduce. I'll try to see if there is anything unusual about my device (for example, maybe because it is managed by organization), but I don't think so. Keeping open to see if others can reproduce - though, I doubt that, something like this would have been fixed quickly if it was common
